crates/control/src/behavior/walk_to_pose.rs

```diff
@@ -166,15 +166,15 @@ impl<'cycle> WalkAndStand<'cycle> {
         let angle_to_walk = target_pose.orientation().angle();
         let was_standing_last_cycle =
             matches!(self.last_motion_command, MotionCommand::Stand { .. });
-        let is_reached = less_than_with_hysteresis(
+        let is_reached = less_than_with_relative_hysteresis(
             was_standing_last_cycle,
             distance_to_walk,
-            self.parameters.target_reached_thresholds.x + self.parameters.hysteresis.x,
+            self.parameters.target_reached_thresholds.x,
```

Comment:
Note to myself, recheck with a drawing, whether that actually does the same

Reply:
I don't think it is. I changed it